README.md

```diff
@@ -138,6 +138,12 @@ boost_where: {user_id: 1} # default factor is 1000
 boost_where: {user_id: {value: 1, factor: 100}}
 ```
 
+Boost indices when searching across more than one index
+
+```ruby
+indices_boost: { Animal.searchkick_index.name => 1, Product.searchkick_index.name => 200 }
```

Comment:
Can we also allow `indices_boost: {Animal: 1, Product: 200}`? We do this with the `index_name` option.